README.md

```diff
@@ -1,77 +1,51 @@
 # Math Magician
 
-This is the first project of the Math Magicians application. I will set up the environment and tools needed to develop a React application. In the following projects, I will develop the actual application.
+This is the first project of the Math Magicians application. I will set up the environment and tools needed to develop a React application. In the following projects, I will develop the actual application. I added, a React component that will hold the core functionality: a calculator.
 
+![Cal](https://user-images.githubusercontent.com/22328716/138008033-5750a9db-e9ed-4684-ba21-45f5b5852a5c.png)
+## Learning objectives
 
-## Setting up my React App
+- Use React components.
+- Use React props.
+- Use React state.
+  
+## Built With
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+- React.js
+  
+## Getting Started
 
-![react](https://user-images.githubusercontent.com/22328716/129612492-73ec21c3-0088-4227-9320-4b01dbf37a84.png)
+To get a local copy up and running follow these simple example steps.
 
-## Available Scripts
+### Setup
 
-In the project directory, you can run:
+- Clone the repo `https://github.com/luqmanmusah/math.git`
+- `cd` into the directory
+- checkout to the interactive branch `git checkout component`
+- Run `npm start` 
+- open in your desired code editor
```

Comment:
- [ ] For this section in the readme file, a user should run `npm install` before running the `npm start` command. Remember that the user needs to grab all project dependencies to be able to run the app successfully.